configuredChains.yaml

```diff
@@ -0,0 +1,103 @@
+## NOTICE: This file gets embedded into ibctest binary
```

Comment:
Should this file be generated from another source, or do we have to hardcode its contents?

Reply:
For lets "hardcode" this. But the idea around this PR is to make this info more transparent and easily editable.